### ISSUE TITLE
Deal with multi class problems

### ISSUE THREAD

**Issue by stephanegaiffas** [pyensemble/wildwood#3]
The `ForestBinaryClassifier` works with multi-class classification problems, but we cannot use #2 for multi-class problems. So, when there are K classes, the strategy is to build K trees instead of 1, which makes n_estimators * K trees total. We need to change labels to one-versus-all for each tree, and predict using normalized score. We can look that https://github.com/scikit-learn/scikit-learn/tree/main/sklearn/ensemble/_hist_gradient_boosting for inspiration

**Comment by stephanegaiffas** [comment]
Fixed in #51
